src/util/PrintUtils.java

```diff
@@ -49,23 +53,23 @@ public static void printStatistic(Map<StatisticKey, Long> statistic) {
     }
 
     public static void writeStatistic(Map<StatisticKey, Long> statistic) {
-        String writePath = new File(new File("").getAbsolutePath()).toString() + File.separator + "solver-statistic.txt";
-        StringBuilder sb = new StringBuilder();
+    	JsonObject obj = new JsonObject();
+    	Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        String writePath = new File(new File("").getAbsolutePath()).toString() + File.separator + "solver-statistic.json";
         for (StatisticKey j : statistic.keySet()) {
             if (statistic.get(j) != (long) 0) {
-                sb.append((j.toString().toLowerCase() + "," + statistic.get(j)) + "\n");
+                obj.addProperty(j.toString().toLowerCase(), statistic.get(j));
             }
         }
         try {
-            File f = new File(writePath);
             PrintWriter pw = new PrintWriter(writePath);
-            pw.write(sb.toString());
+            pw.write(gson.toJson(obj));
```

Comment:
The JsonObject library I used was com.google.gson.JsonObject. The gson object is needed in order to write to a file, since it turns the object into string type. I searched and couldn't find a method that does the same thing in JsonObject. Or a simple obj.toString() could be used, however it will not output pretty json format. Is there anything that you recommend doing here instead? Thanks!